src/vellum/lib/test_suites/resources.py

```diff
@@ -0,0 +1,184 @@
+from __future__ import annotations
+import os
+import time
+from typing import Callable
+
+from src.vellum.lib.test_suites.constants import DEFAULT_MAX_POLLING_DURATION_MS, DEFAULT_POLLING_INTERVAL_MS
+from src.vellum.lib.test_suites.exceptions import TestSuiteRunResultsException
+from src.vellum.types.named_test_case_variable_value_request import NamedTestCaseVariableValueRequest
+from src.vellum.types.test_case_variable_value import TestCaseVariableValue
+from src.vellum.client import Vellum
+from src.vellum.types import (
+    TestSuiteRunExecution,
+    TestSuiteRunMetricOutput,
+    TestSuiteRunState,
+    TestSuiteRunExternalExecConfigRequest,
+    TestSuiteRunExternalExecConfigDataRequest,
+    ExternalTestCaseExecution,
+)
+from src.vellum.lib.utils.paginator import PaginatedResults, get_all_results
+
+
+class VellumTestSuiteRunExecution(TestSuiteRunExecution):
+    @staticmethod
+    def from_api(execution: TestSuiteRunExecution) -> VellumTestSuiteRunExecution:
+        return VellumTestSuiteRunExecution(
```

Comment:
I've also seen this. Unclear to me if one is better than the other

```suggestion
    @classmethod
    def from_api(cls, execution: TestSuiteRunExecution) -> VellumTestSuiteRunExecution:
        return cls(
```

Reply:
Based on Github Copilot's response, I'm switching to classmethod:
<img width="780" alt="Screenshot 2024-04-22 at 2 46 04 PM" src="https://github.com/vellum-ai/vellum-client-python/assets/7143571/db99c1c5-c2c8-4e7e-ad81-47aff8edec66">